theme/icons_test.go

```diff
@@ -67,6 +68,15 @@ func TestNewThemedResource_OneStaticResourceSupport(t *testing.T) {
 	assert.Equal(t, nm, custom.Name())
 }
 
+func TestNewDisabledResource(t *testing.T) {
+	fyne.CurrentApp().Settings().SetTheme(DarkTheme())
+	src := helperNewStaticResource()
+	custom := NewDisabledResource(src)
+	nm := custom.Name()
```

Comment:
I was thinking about commenting on "clr" for color or "rdr" for reader but didn't.
Here, however I would not have guessed that "nm" means name. I don't think we need artificially short variable names - this should be readable at a glance.

Reply:
Yeah, agreed, especially in tests.  Coming from Python, I sometimes go overboard with Go's terse variable naming conventions. Fixed, pushed.